hrjobcontract/js/src/job-contract/controllers/form/form-general.js

```diff
@@ -89,7 +89,7 @@ define([
         months = m.diff(dateStart, 'months');
 
         m.add(-months, 'months');
-        days = m.diff(dateStart, 'days');
+        days = m.diff(dateStart, 'days') + 1;
```

Comment:
handling date is annoying in every language :D 

@MiyaNoctem thanks for trying other options